yt/chyt/controller/internal/strawberry/options.go

```diff
@@ -8,6 +8,7 @@ const (
 	TypeInt64     OptionType = "int64"
 	TypeYson      OptionType = "yson"
 	TypePath      OptionType = "path"
+	TypePoolTrees OptionType = "pool_trees"
```

Comment:
UI shows it as a non-editable string with concated values. Since it cannot be changed it will never be sent back to server, so I don't see how it can break anything,

Reply:
> UI shows it as a non-editable string with concated values. Since it cannot be changed it will never be sent back to server, so I don't see how it can break anything,

Ok. As long as it doesn't break the UI, a new type is perfectly fine